mobile/.flowconfig

```diff
@@ -0,0 +1,75 @@
+[ignore]
```

Comment:
Can we remove this file, since we are not using Flow?

Reply:
Yeah, some files have come from `create-react-native-app`, so I will remove later.